examples/ui/ui.rs

```diff
@@ -3,6 +3,9 @@ use bevy::prelude::*;
 /// This example illustrates the various features of Bevy UI.
 fn main() {
     App::build()
+        .add_resource(WindowDescriptor {
```

Comment:
Why is this needed?

Reply:
It was for testing, forgot to remove it.